phylo/src/evolutionary_models/mod.rs

```diff
@@ -41,6 +43,22 @@ impl Display for DNAModelType {
     }
 }
 
+impl DNAModelType {
+    pub fn get_model_type(model_name: &str) -> Self {
+        match model_name.to_uppercase().as_str() {
+            "JC69" => DNAModelType::JC69,
+            "K80" => DNAModelType::K80,
+            "HKY" => DNAModelType::HKY,
+            "TN93" => DNAModelType::TN93,
+            "GTR" => DNAModelType::GTR,
+            _ => {
+                warn!("Unknown DNA model requested, defaulting to GTR.");
```

Comment:
you probably want to log the type as it was passed, otherwise it might be hard to debug. e.g.
```rust
warn!("Unknown DNA model {model_name} requested. Defaulting to GTR.");
```

Reply:
nit: taking into account how easy it is to return `Result<Self>` and handle it, I would also recommend it instead of using defaults -- let the caller handle it

Reply:
Done.